api/backend/tests/test_service_facade.py

```
@@ -0,0 +1,69 @@
+    polygon = object()
+    expected = (
+        "points",
+        "edges",
+        "pprop",
+        "wprop",
+        "metrics",
+        "access_nodes",
+        "access_edges",
+    )
+
+    async def _fake_graph_from_poly(city_id, polygon):  # type: ignore[override]
```

Ruff (0.14.5)

Unused function argument: `city_id`

(ARG001)